requirements/edx/github.in

```diff
@@ -52,7 +52,6 @@
 
 
 # Python libraries to install directly from github
-git+https://github.com/edx/openedx-calc.git@e9b698c85ad1152002bc0868f475f153dce88952#egg=calc==0.4
 git+https://github.com/edx/openedx-chem.git@ff4e3a03d3c7610e47a9af08eb648d8aabe2eb18#egg=chem==1.0.0
```

Comment:
@feanil: Out of curiosity, was there something about `openedx-calc` that made it slower than `openedx-chem`, or did we just ticket one of these, but not the other?

Reply:
We just ticketed one of these because the other one was done earlier.  We should ticket openedx-chem as well and do the same thing to it.